cppguide.html

```diff
@@ -2171,7 +2222,7 @@ <h3 id="cpplint">cpplint</h3>
 how to run <code>cpplint.py</code> from their project
 tools. If the project you are contributing to does not,
 you can download
-<a href="https://raw.githubusercontent.com/cpplint/cpplint/HEAD/cpplint.py">
+<a href="https://raw.githubusercontent.com/google/styleguide/gh-pages/cpplint/cpplint.py">
```

Comment:
@zetafunction Was there a reason for this change? Since this PR was deployed, this has been linking to the [bankrupted](https://github.com/google/styleguide/pull/837) version...

Reply:
This update is generated from the internal style guide. If someone makes a PR to update the external style guide but that change never makes it to the internal style guide, it's going to be overwritten by the next update.

Reply:
i.e. I screwed up.  someone has posted a fix internally for it though, so it should make its way back out.